src/main/java/uk/gov/ons/ctp/integration/contactcentresvc/service/impl/CaseServiceImpl.java

```diff
@@ -304,7 +304,21 @@ public CaseDTO getCaseById(final UUID caseId, CaseQueryRequestDTO requestParamsD
 
     List<CaseDTO> ccsCases = new ArrayList<CaseDTO>();
     for (CaseContainerDTO ccsCaseDetails : ccsCaseContainersList) {
-      ccsCases.add(caseDTOMapper.map(ccsCaseDetails, CaseDTO.class));
+      // Decide if the case type indicates it's worth returning the case
+      boolean isSupportedCaseType = false;
+      String caseType = ccsCaseDetails.getCaseType();
+      if (caseType != null) {
+        isSupportedCaseType =
+            caseType.equals("HH") || caseType.equals("CE") || caseType.equals("SPG");
```

Comment:
Should avoid static strings for caseType. 
caseType.equals(CaseType.HH.name()) etc

Reply:
Agree.
Code updated